READMEs/get-ocean-on-sapphire.md

```diff
@@ -0,0 +1,45 @@
+# Get OCEAN tokens on Sapphire Mainnet
+
```

Comment:
Add a section at the top, e.g. `## 0. Steps`.

This section should list the high-level steps:
1. Get OCEAN on Eth mainnet
2. Transfer OCEAN  to Sapphire mainnet via Celer
3. Verify

Each of those steps should have a url linking to the appropriate section.

Each section header should be numbered accordingly. Eg `# 1. Get OCEAN on Eth Mainnet`.

The steps within each section should be numbered. E.g. `# 1.1 Create a wallet`.

Notes:
- you don't need to say "OCEAN tokens", just say "OCEAN". Uppercase implies the token, therefore "token" is redundant
- wherever you can reduce the # characters without hurting understandability, you could. Why: it enhances readability because it's shorter. That's why I changed "Ethereum mainnet" --> "Eth mainnet". These people are technical, they'll get it

Reply:
Don't bother with sub-headings in the TOC. It's too hard to maintain. Other than that, looks great!